webui/react/src/hooks/useModal/Model/useModalModelCreate.tsx

```diff
@@ -155,14 +155,14 @@ const useModalModelCreate = (): ModalHooks => {
         </p>
         <div>
           <h2>Model name</h2>
-          <Input value={modelName} onChange={updateModelName} />
```

Comment:
I found a solution without react hook form.
All I need to do is to remove the value from props